src/red_commons/logging/__init__.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import contextlib
+import logging
+
+VERBOSE = logging.DEBUG - 3
+TRACE = logging.DEBUG - 5
+
+__all__ = [
+    "RedTraceLogger",
+    "VERBOSE",
+    "TRACE",
+    "update_logger",
+    "task_callback_critical",
+    "task_callback_exception",
+    "task_callback_warning",
+    "task_callback_debug",
+    "task_callback_verbose",
+    "task_callback_trace",
+]
+
+
+class RedTraceLogger(logging.getLoggerClass()):
```

Comment:
This is better supported by some linters and type checkers and it's going to give us the same class considering how early we will be importing this module.
```suggestion
class RedTraceLogger(logging.Logger):
```